src/main/webapp/app/shared/user-settings/ide-preferences/ide-settings.component.html

```diff
@@ -0,0 +1,72 @@
+<h1>
+    <!--Default IDE Settings-->
+    {{ 'artemisApp.userSettings.idePreferences' | artemisTranslate }}
```

Comment:
We prefer the translate directive `jhiTranslate`

Reply:
does not work with jhiTranslate as it does not take the heading style correcly

Reply:
nvm i just have to but it inside of the heading tag